configs/beaker_configs/default_finetune_lora_multinode.yaml

```diff
@@ -5,6 +5,9 @@ tasks:
     replicas: 4
     leaderSelection: true
     hostNetworking: true
+    propagateFailure: true
+    propagatePreemption: true
+    synchronizedStartTimeout: 30m
```

Comment:
This seems like a very generous timeout. What about setting it to like 5 mins?

Reply:
5 mins might be too short as one replica might be waiting the resources for get started. Let me set this to 15 mins (the recommended value in beaker doc).